lib/pg_index_benchmark/runner/benchmark.rb

```diff
@@ -284,28 +295,45 @@ def run_query_for_scenario(scenario, query_text, format = :json)
               raw_plan(query_text)
             end
           )
-        @result_plans[fingerprint] = {} unless @result_plans.key?(fingerprint)
-        @result_plans[fingerprint][scenario] = plan
+        @all_query_plans_by_query[fingerprint] = {
+        } unless @all_query_plans_by_query.key?(fingerprint)
```

Comment:
should not be on one line ?

Reply:
I double checked, this new line is caused by stree when linting the file :)